portality/migrate/p1p2/country_cleanup.py

```diff
@@ -4,11 +4,12 @@
 import csv
```

Comment:
Realistically, I'm not sure that this file would ever run or be run again.  Is there an argument for just deleting these directories when they become obsolete?  They will be in version control if we ever needed them.  Or we could leave them broken deliberately to record what it was that they looked like at the time they were run?